config/routes.rb

```diff
@@ -0,0 +1,10 @@
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # get '/users', to: 'users#index'
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :show]
+  end
+    #   resources :comments, only [:new, :create]
+    #   resources :likes, only [:create]
```

Comment:
- [ ] @BilalLiaquat7 Great work implementing the changes requested, however, you forgot to remove the commented code in your codebase, it's not good practice to have commented code in your codebase.